app/src/main/res/layout/settings_page.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#FCFCFCFC">
+
+    <!--Top Menu-->
+
+    <Button
+        android:id="@+id/done2"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="18dp"
+        android:layout_marginStart="310dp"
+        android:fontFamily="@font/roboto_bold"
+        android:text="Done"
+        android:textAllCaps="false"
+        android:textColor="#4F4F4F"
+        android:textSize="24sp" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="200dp"
+        android:layout_height="60dp"
+        android:layout_marginStart="104dp"
+        android:layout_marginTop="20dp"
+        android:contentDescription="toggle"
+        app:srcCompat="@drawable/ic_option_pill_bar2" />
+
+    <Button
+        android:id="@+id/toggle2_1"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="60dp"
+        android:layout_height="44dp"
+        android:layout_marginStart="110dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="2dp"
+        android:backgroundTint="#00FFFFFF"
+        android:visibility="visible"/>
+
+    <Button
+        android:id="@+id/toggle2_2"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="60dp"
+        android:layout_height="44dp"
+        android:layout_marginStart="240dp"
+        android:layout_marginTop="22dp"
+        android:backgroundTint="#00FFFFFF"
+        android:visibility="visible" />
+
+    <!--My preferences-->
+
+    <TextView
+        android:id="@+id/myPreferences"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="#282828"
+        android:backgroundTint="#00FFFFFF"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="90dp"
+        android:textSize="20sp"
+        android:text="My preferences"/>
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="115dp"
+        android:background="#000000"/>
+
+    <CheckBox
```

Comment:
check boxes should be round 
checked
![checked](https://user-images.githubusercontent.com/22159403/77816397-d048e700-7098-11ea-821e-da98b60c0a48.png)
unchecked
![unchecked](https://user-images.githubusercontent.com/22159403/77816398-d0e17d80-7098-11ea-9160-e59adf98456e.png)